packages/nimble-components/src/anchor/tests/anchor.stories.ts

```diff
@@ -33,15 +32,15 @@ const metadata: Meta<AnchorArgs> = {
     // prettier-ignore
     render: createUserSelectedThemeStory(html`
         <style class='code-hide'>
-            * {
+            span {
```

Comment:
I made this change so that the `*` styling wouldn't mess with the styling of the mdx docs.